parse_test.go

```diff
@@ -339,19 +338,23 @@ type ParserData struct {
 	GpgSignature []byte
 }
 
-// filterNondeterministicData replaces some generated fields with static strings
-func filterNondeterministicData(data ParserData) ParserData {
+// filterData replaces some generated fields with static strings
+func filterData(data ParserData) ParserData {
 	copiedData := maps.Clone(data.Metadata)
 
 	fields := []string{
 		"blake2_256_digest",
 		"md5_digest",
 		"sha256_digest",
+		"version",
 	}
 
 	for _, field := range fields {
-		if _, ok := copiedData[field]; ok {
-			copiedData[field] = []string{fmt.Sprintf("%s exists", field)}
+		if value, ok := copiedData[field]; ok {
+			values := lo.Map(value, func(_ string, _ int) string {
+				return fmt.Sprintf("%s exists", field)
+			})
+			copiedData[field] = values
```

Comment:
Should this be simplified back to what it was?
```suggestion
			copiedData[field] = []string{fmt.Sprintf("%s exists", field)}
```

Reply:
I don't think so. It's unlikely, but since the input is a list of strings we don't want to assume the length. For example if we somehow send a list of versions or checksums we'd want to know.